# goal
bool ifDetect # always true for starting perception

---
# result

bool success
string message # error message
geometry_msgs/Pose pose # pose of the detected beams center
---

# feedback
float32 progress # progress of the perception, 0.0 to 1.0
